### PR TITLE
[Feature] Add Localizable.strings for 13 supported languages

### PR DESCRIPTION
**Pull request by jimchou-dev**
## Purpose

* Enable localization setting for the project
* Add Localizable.strings for the following 13 languages:
  * Chinese-Simplified
  * Spanish
  * Russian
  * Japanese
  * French
  * Portuguese
  * German
  * Korean
  * Italian
  * Chinese-Traditional
  * Dutch
  * Turkish
  * English (UK)

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
Verify that the following are valid
* Check in project that Localization has these 13 languages + English

## Other Information

* These strings are not currently used yet
* Translated string does not have trailing spaces, so BannerTextView will need to accommodate for adding the spaces in the compliance banner

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,66 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+/* Devices */
+"AzureCommunicationUI.AudioDevice.iOS"="iOS";
+"AzureCommunicationUI.AudioDevice.Speaker"="Lautsprecher";
+"AzureCommunicationUI.AudioDevice.iPhone"="iPhone";
+"AzureCommunicationUI.AudioDevice.iPad"="iPad";
```
Comment (AmyL219) [inline AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings:10]: Do those keys consistent with Android side? For example, they would have `azure_communicaton_ui_audio_device_android`. 
I wonder to match our naming convenient, should we rename them into `AzureCommunicationUI.Common.AudioDevice.iOS` to match `AzureCommunicationUI.{​​​​​​​View}​​​​​​​.{​​​​​​​Component}​​​​​​​.{​​​​​​​Identifier}`

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: Report a bug
```
[the remark this answers is not in the document]
Comment (jimchou-dev) [reply .github/ISSUE_TEMPLATE/report-a-bug.md:2]: I think I may have branch off from main by accident, let's back merge main into develop/beta and this should be included. I'll create a separate PR for that

File: AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (jimchou-dev) [reply AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings]: Have synced with android on the keys, submitted new translation request to touchdown service

File: AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+/* SetupView */
+"AzureCommunicationUI.SetupView.JoinCall"="An Anruf teilnehmen";
```
[the remark this answers is not in the document]
Comment (jimchou-dev) [reply AzureCommunicationUI/AzureCommunicationUI/Localization/de.lproj/Localizable.strings]: Have synced with android on the keys, submitted new translation request to touchdown service

**Review by AmyL219 — APPROVED** [review]

**Pull request merged**